runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -1269,6 +1286,7 @@ MM_IncrementalGenerationalGC::partialGarbageCollectUsingCopyForward(MM_Environme
 
 	/* Record stats before a copy forward */
 	UDATA freeMemoryForSurvivor = _extensions->getHeap()->getActualFreeMemorySize();
+	env->_heapSizingData.freeTenure = freeMemoryForSurvivor;
```

Comment:
In general, this should be reduced by eden size, right? 
But at this point eden is full, so this does happen to be correct? If so, put a comment above.

Reply:
Also should we reduce by reserved size (although it's likely done later at appropirate points when ths is used)

We should also try to avoid using work 'tenure' word in Balanced as much as possible, as it's already the case in the code (we don't have tenure on physical level), but if we do we should define variable (comment at declaration point) what we mean by that (for example, free memory excluding eden (and survivor reserved regions?)). 

Or should we just call it nonEden? It's already commonly used, although for live/used memory separation.

Reply:
Exactly yes eden is full here. I will add a comment.

Reply:
I like the idea of just adding comment at declaration points. I think if it's called `nonEden`, it might come across to a reader as not including survivor space. 

Maybe just calling it (albeit more verbose) `freeNonEdenAndSurvivor` - thoughts?